fluidBuild.config.cjs

```diff
@@ -551,4 +551,14 @@ module.exports = {
 		"release/**": "patch",
 		"next": "major",
 	},
+
+	releaseNotes: {
+		sections: [
+			{ name: "feature", heading: "✨ New Features" },
```

Comment:
(Not necessarily for this PR) I wonder if we also want `build` for dependency changes, and `docs` for documentation changes.

Reply:
I think this list may be different every release. Part of the motivation for making it completely config based is to
give the release notes owner the ability to organize things on the fly (see my 2.2 release notes PR).